Streetcode/Streetcode.BLL/Services/BlobStorageService/BlobService.cs

```diff
@@ -64,17 +69,27 @@ public void SaveFileInStorageBase64(string base64, string name, string extension
         EncryptFile(imageBytes, extension, name);
     }
 
-    public void DeleteFileInStorage(string name)
+    public void DeleteFileInStorage(string? name)
     {
+        if (name is null)
+        {
+            throw new ArgumentNullException("Blob name is null.");
```

Comment:
If you don't need custom exception message consider using oneliner `ArgumentNullException.ThrowIfNull(name)`